README.md

```diff
@@ -25,36 +34,7 @@ yarn add @crossmint/client-sdk-react-ui
 
 ### 2. Set up
 
-On your main App file (e.g. app.tsx):
-
-1. Require the CrossMint styles by adding `require('@crossmint/client-sdk-react-ui/styles.css');` 
-2. Import `CrossMintProvider`, wrap your app with it at the top level and pass the clientId you received after filling in [the onboarding form](https://www.crossmint.io/developers/onboarding).
-
-```javascript
-import { AppProps } from "next/app";
-
-import { CrossMintProvider } from "@crossmint/client-sdk-react-ui";
-
-// Use require instead of import, and order matters
-require("../styles/globals.css");
-require("@crossmint/client-sdk-react-ui/styles.css");
-
-export default function App({ Component, pageProps }: AppProps) {
-    return (
-        <CrossMintProvider clientId="<YOUR_CLIENT_ID>">
-            <Component {...pageProps} />
-        </CrossMintProvider>
-    );
-}
-```
-
-### 3. Usage
-
-Go to the main file where your Candy Machine button lives. For example, Home.tsx.
-
-There, import the Pay with `CrossMintButton`, and add it in the UI. 
-
-**Important**: be sure to test that the Crossmint button is visible even if a user didn't connect their wallet! Else, your users without wallets won't be able to use it.
+Just import the the Pay with `CrossMintButton` into your app wherever you would like. It's that simple!
```

Comment:
Let's add back a point to ensure they test that the button shows up if a user doesn't have a crypto wallet like Phantom. A lot of ppl make this mistake where they put it in a place where it's not visible unless you have a wallet connected